plugins/harvesters/oai/OAIHarvester.inc.php

```diff
@@ -201,6 +201,8 @@ function getSets($harvesterUrl) {
 			return false;
 		}
 
+		if (!$result) return false;
```

Comment:
Hi @asmecher, had a peek at https://www.openarchives.org/OAI/openarchivesprotocol.html#ListSets and it looks like 'no sets' should be returned via an error code (which we check for separately below). The above check still seems necessary though in case there's no response to the request.